Lib/test/test_tools/test_i18n.py

```diff
@@ -517,49 +517,45 @@ def test_parse_keyword_spec(self):
                     parse_spec(spec)
                 self.assertEqual(str(cm.exception), message)
 
-    def test_normalize_multiline(self):
+    def setUp(self):
         # required to set up normalize
-        options = SimpleNamespace(width=78)
         make_escapes(True)
 
+    def test_normalize_multiline(self):
         s = 'multi-line\n translation'
         s_expected = '""\n"multi-line\\n"\n" translation"'
 
-        data = normalize(s, 'UTF-8', 'msgid', options)
+        data = normalize(s, 'UTF-8', 'msgid', 78)
         self.assertEqual(s_expected, data)
 
     def test_normalize_wrap(self):
-        # required to set up normalize
-        options = SimpleNamespace(width=30)
-        make_escapes(True)
-
-        s = 'this string should be wrapped to 30 chars'
-        s_expected = '""\n"this string should be "\n"wrapped to 30 chars"'
-
-        data = normalize(s, 'UTF-8', 'msgid', options)
-        self.assertEqual(s_expected, data)
+        cases = (
+            ('multi-line\n translation', '""\n"multi-line\\n"\n" translation"'),
+            ('fee fi fo fum fee fi ', '"fee fi fo fum fee fi "'),         # len = 29
+            ('fee fi fo fum fee fi f',  '"fee fi fo fum fee fi f"'),      # len = 30
+            ('fee fi fo fum fee fi fo', '""\n"fee fi fo fum fee fi fo"' ),# len = 31
+        )
+        for raw, expected in cases:
+            with self.subTest(raw):
+                data = normalize(raw, 'UTF-8', 'msgid', 30)
+                self.assertEqual(expected, data)
 
     def test_normalize_nostr(self):
-        # required to set up normalize
-        options = SimpleNamespace(width=30)
-        make_escapes(True)
-
-        s = ''
-        s_expected = '""'
-
-        data = normalize(s, 'UTF-8', 'msgid', options)
-        self.assertEqual(s_expected, data)
-
-    def test_normalize_short_width(self):
-        # required to set up normalize
-        options = SimpleNamespace(width=3)
-        make_escapes(True)
-
-        s = 'foos'
-        s_expected = '"foos"'
-
-        data = normalize(s, 'UTF-8', 'msgid', options)
-        self.assertEqual(s_expected, data)
+        data = normalize('', 'UTF-8', 'msgid', 30)
+        self.assertEqual('""', data)
+
+    def test_normalize_single_word(self):
+        for s in ("fee", "fi", "fo", "fums"):
+            data = normalize(s, 'UTF-8', 'msgid', 3)
```

Comment:
Would not be clearer to test the same string with different widths? 3 is less than the length of the prefix "msgid", so the input is wrapped in any case.

Why the case of a single word needs a test?